comp/core/workloadfilter/program/cel_program.go

```diff
@@ -54,14 +37,18 @@ func (p CELProgram) Evaluate(entity filterdef.Filterable) (filterdef.Result, []e
 					return filterdef.Excluded, nil
 				}
 			} else {
-				errs = append(errs, fmt.Errorf("exclude (%s) result not bool: %v", p.Name, out.Value()))
+				log.Criticalf(`filter '%s' from 'cel_workload_exclude' failed to convert value to bool: %v`, p.Name, out.Value())
+				log.Flush()
+				os.Exit(1)
 			}
 		} else {
-			errs = append(errs, fmt.Errorf("exclude (%s) eval error: %w", p.Name, err))
+			log.Criticalf(`filter '%s' from 'cel_workload_exclude' failed to convert evaluate: %v`, p.Name, out.Value())
+			log.Flush()
+			os.Exit(1)
 		}
 	}
 
-	return filterdef.Unknown, errs
+	return filterdef.Unknown, nil
```

Comment:
nit: There doesn't appear to be any `Evaluate` implementations that return an error after this change. Should we consider removing `[]error` from the interface definition?